### PR TITLE
Alter recommendation decision for experiments that are assigned upfront

### PR DESCRIPTION
**Pull request by jessie-ross**
**This PR:**
- Introduces the idea of experiments assigned up front.
- Ignores the time based warning for these experiments.

## How has this been tested?

- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)

### CODE REVIEW

File: src/lib/experiments.test.ts [hunk 1]
```diff
@@ -1,50 +1,59 @@
 import Fixtures from 'src/test-helpers/fixtures'
```
Comment (jessie-ross) [inline src/lib/experiments.test.ts:1]: Mostly whitespace changes here, not sure why they occurred.

**Comment by yanirs** [comment]
@jessie-ross can you please provide more context and an explanation of what this PR does? The assumption that email experiments are all assigned at once isn't necessarily correct, as some email experiments are for drip campaigns.

**Comment by jessie-ross** [comment]
> @jessie-ross can you please provide more context and an explanation of what this PR does? The assumption that email experiments are all assigned at once isn't necessarily correct, as some email experiments are for drip campaigns.

Ahh I thought that was the case, you mention in https://github.com/Automattic/abacus/issues/419#issuecomment-758314643:
> duration warnings are irrelevant for email experiments.

**Comment by yanirs** [comment]
Yeah, I was wrong because I was only thinking of mass emails that typically run over a short period. It may be better to keep things simple and _not_ suppress the time-based warnings based on the platform, as experimenters are free to ignore warnings anyway.

**Pull request closed without merging**